src/Client.ts

```diff
@@ -49,6 +49,7 @@ export type ListMessagesOptions = {
   startTime?: Date
   endTime?: Date
   limit?: number
+  direction?: messageApi.SortDirection
```

Comment:
Do we want to expose the `messageApi.SortDirection` type? It feels a bit cumbersome, given that this could basically be a boolean.

Reply:
We already expose it on the paginated endpoint

Reply:
Hm, worth fixing that too?

Reply:
That would be a breaking API change and require a major version update. We certainly can if we feel strongly about it

Reply:
Could also make a backlog item and roll it in to the next major version update

Reply:
I don't feel strongly about it, if you agree then sounds good, if not I'm fine leaving it as is.

Reply:
I can see how it's a nicer API. I'd be fine rolling it into the next major version update.